docs/Changelog.md

```diff
@@ -14456,7 +14457,7 @@ This version of the operator has been available since version 12 of the default
 
 <dl>
 <dt><tt>X</tt> : T</dt>
-<dd>Input tensor</dd>
+<dd>Input tensor. Must be full-rank.</dd>
```

Comment:
nit: To be consistent, let's say `The matrix must be invertible.`

Reply:
How about "Every matrix in the batch must be invertible"?